crates/dojo-core/src/utils/utils.cairo

```diff
@@ -0,0 +1,103 @@
+use core::option::Option;
+
+use dojo::model::{Layout, FieldLayout};
+
+/// Compute the poseidon hash of a serialized ByteArray
+pub fn hash(data: @ByteArray) -> felt252 {
```

Comment:
We may rename this function as `hash` is very generic.

`bytearray_hash` could make it more specific?

Reply:
Yes, good idea 👍